src/main/java/net/wurstclient/altmanager/MojangAlt.java

```diff
@@ -13,10 +13,10 @@
 
 import net.wurstclient.WurstClient;
 
-public final class MojangAlt extends Alt
+public class MojangAlt extends Alt
 {
-	private final String email;
-	private final String password;
+	public String email;
+	public String password;

```

Comment:
These fields are not meant to be mutable or public. As far as I can tell you're not actually changing or reading them externally, so it should be safe to just change the modifiers back to `private final`.

Reply:
Will do.